app/controllers/registrations_controller.rb

```diff
@@ -40,6 +40,7 @@ def create
   def update_first_user_permissions(resource)
     return unless Settings::General.waiting_on_first_user
 
+    resource.add_role(:forem_owner)
```

Comment:
I would call this `:creator`, not `:owner`, not `:forem_owner`. For mainly two reasons:

- we call them creators anywhere else, introducing the concept of "owner" means having a disconnect in the domain language, and we already have a few of those (member vs user, post vs article, listing vs classified listing and so on). Creator is the language the Community Success team speaks, the word Product and Design use and so on. Unless this is not the creator and it's an entirely different role for a different purpose 🗡️ 
- I would remove `forem_` as a prefix because it's redudant, they can't be the owner/creator of anything else ;-)

Reply:
I was basing this of the conversations on Slack yesterday as the owner could change over time, but happy to put it back to `creator`. And I agree the `forem_` prefix is redundant. Thanks for the feedback!

Reply:
Waiting for Jamie and Macs thoughts on the above but I'm in favour of `creator` for all the reasons that Rhymes has listed.